.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: java
```

Comment:
Enabling Travis could be a separate PR

Reply:
True, but it's simple enough to have it as a base, even if it's only for running tests and jacoco.